docs/components/console/manage-clusters/update-cluster.md

```diff
@@ -0,0 +1,21 @@
+---
+id: update-cluster
+title: Update your cluster
+description: "Clusters eligible for updates will show a button on the UI."
```

Comment:
I think I would change this as to _why_ someone might want to update their cluster.

Reply:
Adjustment in the upcoming commit.